osu.Desktop/osu.Desktop.csproj

```diff
@@ -14,6 +14,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <StartupObject>osu.Desktop.Program</StartupObject>
+    <IlcOptimizationPreference>Speed</IlcOptimizationPreference>
```

Comment:
Again there were no issues and research done to see how much this would improve the project all in all (no test benchmarks either). Do not just add things without raising an issue regarding changes like this.

Reply:
Do not talk with this kind of authority on the project please, it sounds shocking and incorrect.